hack/go-linter.sh

```diff
@@ -12,7 +12,7 @@ golangci-lint run --disable-all \
     --enable=maligned \
     --enable=ineffassign \
     --enable=goconst \
-    --enable=lll \
+    --enable=lll -e "internal value || yum "\
```

Comment:
Could you please let us know why it is required? Also, could you please add a comment in the code with as remove the `enable= lll` from the commented code?

Reply:
there are lines of code that if `--enable=lll` is enabled they do not run because they lines cannot be modified https://github.com/I-am-Dishon/operator-sdk/blob/master/internal/scaffold/ansible/dockerfilehybrid.go#L68

Reply:
according to the `lll` documentation you can exclude lines with `-e` flags. If there is  a better suggestions to go about this  am willing to take them into consideration

Reply:
Could you please add a comment with why it was added in the code and its reference? Something as for example:

```suggestion
     # The following arg ` -e "internal value || yum "` was added to ignore any line
     # which has the text `internal value`  or `yum`. More info: <URL doc ref>
    --enable=lll -e "internal value || yum "\
```
PS.: In this case use --exclude instead of --e shows that make more clear what it is doing.   

Also, see that you can ignore the lint when/if required with the comment `//nolint:lll`.